src/IceRpc.Transports.Coloc/Internal/ColocListener.cs

```diff
@@ -56,6 +56,7 @@ internal class ColocListener : IListener<IDuplexConnection>
         catch (OperationCanceledException)
         {
             cancellationToken.ThrowIfCancellationRequested();
+            ObjectDisposedException.ThrowIf(_disposeCts.IsCancellationRequested, this);
```

Comment:
My understanding is:
 - we get OCE only when cts is canceled
 - cts can be canceled if cancellationToken is canceled (=> we throw on the line above) OR
   _disposeCts is canceled (=> 'this' was disposed). It can't be canceled for any other reason


As a result, we can never reach line 60. It would be much clearer to write this as:
```cs
cancellationToken.ThrowIfCancellationRequested();
Debug.Assert(_disposeCts.IsCancellationRequested);
throw new ObjectDisposedException(...); // comment on why we do this
```